src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    text-align: center;
+    margin-bottom: 36px;
+  }
+
+  &__logo {
+    height: 134px;
+    width: 160px;
+  }
+  
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 5px;
+  }
+  
+  &__id {
+    font-weight: 400;
+    font-size: 10px;
+    color: #616070;
+    margin-bottom: 17px;
+  }
+  
+  &__feedback {
+    display: flex;
+    justify-content: space-between;
```

Comment:
I would suggest creating placeholder for flex and space-beetwen. I have seen these styles repetead for several times.